angr/sim_type.py

```diff
@@ -801,6 +811,7 @@ def make_preamble():
 
         if isinstance(typ, (SimTypeNum, SimTypeInt)) and str(typ) not in BASIC_TYPES:
             try:
+                # TODO: Investigate whether this needs to be re-imagined using byte_width
```

Comment:
it does, but if anyone tries to use the type parsing infrastructure with nonstandard byte widths and then complains to me that it doesn't work I will personally strangle them

Reply:
I'm happy with leaving that be as well